test/jdk/jdk/incubator/vector/templates/Unit-Miscellaneous.template

```diff
@@ -633,14 +625,22 @@
     static void ShapeWithLanes$vectorteststype$SmokeTest() {
         $abstractvectortype$ av = $abstractvectortype$.zero(SPECIES);
         VectorShape vsh = av.shape();
+#if[Halffloat]
+        VectorSpecies species = vsh.withLanes(Halffloat.class);
+#else[Halffloat]
         VectorSpecies species = vsh.withLanes($type$.class);
+#end[Halffloat]
```

Comment:
You can save adding special handling in template by introducing a new template-variable